doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -9,42 +9,74 @@ How to customize the SUIT DFU process
    :local:
    :depth: 2
 
-Nordic Semiconductor provides a SUIT sample (``nrf54h_suit_sample``) in which the build system uses predefined configuration.
-The specified Kconfig options in the sample can be used to modify the artifacts produced by the build system, customize the DFU process, and integrate the DFU solution with external build systems.
+Nordic Semiconductor provides a Software Update Internet of Things (SUIT) sample (```nrf54h_suit_sample``) which uses predefined configurations in the build system.
+The specified Kconfig options in the sample can be used to customize the DFU process and integrate the DFU solution with external build systems.
+This guide provides a comprehensive set of instructions for modifying values in the :ref:`SUIT manifest <ug_nrf54h20_suit_manifest_overview>`.
 
-.. note::
-    In order to customize the SUIT DFU process, we strongly recommend that you modify the values in the manifest that are associated with the ID values that are configured into the device.
-    This includes both the ``class-identifier`` and ``vendor-identifier`` values found in manifest(s).
+Understanding SUIT DFU
+**********************
```

Comment:
Consider changing the subheading for the consistency with other docs and the image following the text here. Suggestions:
"Overview" or "Overview of the SUIT DFU procedure"

Reply:
Applied